knowledge_repo/app/routes/render.py

```diff
@@ -100,6 +100,11 @@ def render():
 
     is_author = user_id in [author.id for author in post.authors]
 
+    web_editor_prefixes = current_app.config['WEB_EDITOR_PREFIXES']
+    is_webeditor = []
+    if web_editor_prefixes:
+        is_webeditor = [prefix for prefix in web_editor_prefixes if prefix in path]
```

Comment:
`prefix in path` is a little too loose here, since sometimes a prefix could appear within the path at some other location than the beginning; e.g.: `projects/webposts/why_you_should_use_git.kp`.

Given that you are using it as a boolean indicator, perhaps you could simplify the code here by just doing:

```
show_webeditor_buttons = any(prefix.startswith(path) for prefix in current_app.config['WEB_EDITOR_PREFIXES'])
```